addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -168,4 +175,16 @@
 		<description>Media type of the current file</description>
 		<state readOnly="true" pattern="%s"></state>
 	</channel-type>
+	<channel-type id="thumbnail" advanced="true">
+		<item-type>String</item-type>
```

Comment:
I think if everybody always only creates workarounds for missing features, we will never see support in HABPanel and other UIs for it. So I would suggest to ask HABPanel to support image items (which should not be hard at all to implement).